data/brands/shop/supermarket.json

```
@@ -4745,6 +4745,17 @@
         "shop": "supermarket"
       }
     },
+    {
+      "displayName": "Metro Supermarket"
+      "locationSet": {"include": ["ph"]},
```

```suggestion
      "locationSet": {"include": ["ph"]},
      "matchNames": ["metro gaisano"],
```

`matchNames` should be here, not inside `tags`